api/src/AppBundle/v2/Assembler/UserAssembler.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+
+namespace AppBundle\v2\Assembler;
+
+class UserAssembler
```

Comment:
Is this empty class required for something?

Reply:
Ah good spot - no I was making this while trying to fix some missing data coming back from the API. I'll drop it out.